watertap/unit_models/zero_order/ozone_aop_zo.py

```diff
@@ -29,31 +29,58 @@ class OzoneAOPZOData(OzoneZOData):
     Zero-Order model for a Ozone-AOP unit operation.
     """
 
-    CONFIG = OzoneZOData.CONFIG()
-
     def build(self):
         super().build()
 
         self._tech_type = "ozone_aop"
 
-
         self.oxidant_dose = Var(self.flowsheet().time,
-                                units=pyunits.mg/pyunits.L,
+                                units=pyunits.kg/pyunits.m**3,
                                 doc="Oxidant dosage")
 
-        self.chemical_flow_mass = Var(self.flowsheet().time,
-                                    units=pyunits.kg/pyunits.s,
-                                    bounds=(0, None),
-                                    doc="Mass flow rate of oxidant solution")
+        self.chemical_flow_mass = Var(
+            self.flowsheet().time,
+            units=pyunits.kg/pyunits.s,
+            bounds=(0, None),
+            doc="Mass flow rate of oxidant solution")
+
+        self.ozone_toc_ratio = Var(self.flowsheet().time,
+                                   units=pyunits.dimensionless,
+                                   doc="Ratio of ozone to total organic carbon")
 
-        self._fixed_perf_vars.append(self.oxidant_dose)
+        self.hydrogen_peroxide_ozone_ratio = Var(self.flowsheet().time,
+                                                 units=pyunits.dimensionless,
+                                                 doc="Ratio of hydrogen peroxide to ozone")
+
+        self._fixed_perf_vars.append(self.hydrogen_peroxide_ozone_ratio)
+
+        @self.Constraint(self.flowsheet().time,
+                         doc="Ozone/TOC ratio constraint")
+        def ozone_toc_ratio_constraint(b, t):
+            return (b.ozone_toc_ratio[t] == 1
+                    + b.concentration_time[t] / b.contact_time[t]     #TODO: think this should be dividing by contact time, not multiplying
+                    / pyunits.convert(b.properties_in[t].conc_mass_comp['toc'],
```

Comment:
I would also put the entire expression inside this `convert` statement:

```
pyunits.convert(conc_time/contact_time/conc_mass_comp, to_units=dimensionless)
```

This will also help make sure the units in this expression are right,